TOC-tidb-cloud.md

```diff
@@ -53,6 +53,7 @@
     - Ruby
       - [mysql2](/develop/dev-guide-sample-application-ruby-mysql2.md)
       - [Rails](/develop/dev-guide-sample-application-ruby-rails.md)
+    - [WordPress](/develop/dev-guide-wordpress.md)
```

Comment:
Most existing filenames for this section are in this format: `dev-guide-sample-application-xxx.md`. Do we need to keep the filename style consistent?

Reply:
This is not about building a sample app. So we are good. :)